controls/treeview/item-template.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Item Template
+page_title: .NET MAUI TreeView Documentation - Item Template
+description: Review how to customize the default text displayed in the Telerik .NET MAUI TreeView control by using the item template. 
+position: 7
+slug: treeview-item-template
+---
+
+# .NET MAUI TreeView Item Template
+
+Customize the text displayed in the TreeView item by using the `ItemTemplate` property. The `ItemTemplate` applies on TreeView level. The `ItemTemplate` does not include the checkbox, image and expand/collapse indicator. These elements are part of the `ControlTemplate`. For more details about how to customize the TreeView Control Template review the [Control Template]({%slug treeview-control-template%}) article. 
+
+Here is an example how to define custom `ItemTemplate`: 
```

Comment:
The following example shows how to do thing1, thing2, thing3 by using the `ItemTemplate` property.